metricbeat/module/elasticsearch/node_stats/data.go

```diff
@@ -257,6 +256,21 @@ var (
 			"get":    c.Dict("get", threadPoolStatsSchema),
 			"search": c.Dict("search", threadPoolStatsSchema),
 		}),
+		"indexing_pressure": c.Dict("indexing_pressure", s.Schema{
+			"memory": c.Dict("memory", s.Schema{
+				"current":        c.Dict("current", currentMemoryPressureSchema),
+				"total":          c.Dict("total", totalMemoryPressureSchema),
+				"limit_in_bytes": c.Int("limit_in_bytes"),
+			}),
+		}),
+		"ingest": c.Dict("ingest", s.Schema{
```

Comment:
Looks like this still needs fields entries

Reply:
@matschaffer as in, these fields need to be entered into the `fields.yml` file?

Reply:
yep! the integration tests are currently failing with:

```
Exception: Key 'elasticsearch.node.stats.ingest.total.count' found in event ... is not documented!
```

You should be able to confirm via https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats/detail/PR-28479/20/pipeline